openapi/pandadoc/Package.md

```diff
@@ -0,0 +1,18 @@
+Connects to [PandaDoc API](https://developers.pandadoc.com/reference/about) from Ballerina
+
+### Package overview
+The `pandadoc` is a [Ballerina](https://ballerina.io/) connector for connecting to PandaDoc API. This package allows you to programmatically interact with a PandaDoc account.
```

Comment:
Shall we replace this line with
```
The `ballerinax/pandadoc` is a [Ballerina](https://ballerina.io/)
```
to maintain consistency with other connector Package.md

Reply:
Updated